bee-protocol/src/workers/propagator.rs

```diff
@@ -62,14 +62,11 @@ async fn propagate<B: StorageBackend>(
                     .await
                 {
                     Some(parent_sepi) => (IndexId::new(parent_sepi, *parent), IndexId::new(parent_sepi, *parent)),
-                    None => match tangle.get_metadata(parent).await {
-                        // 'unwrap' is safe (see explanation above)
-                        Some(parent_md) => (
-                            parent_md.omrsi().expect("solid msg with unset omrsi"),
-                            parent_md.ymrsi().expect("solid msg with unset ymrsi"),
-                        ),
-                        None => continue 'outer,
```

Comment:
Is the new code really equivalent to the older version? It seems like the old code was skipping later statements due to `continue 'outer`.

Reply:
It is not equivalent but it was really weird. Failing to load the metadata would have been silent after discovering that a message should be set solid but before actually marking it as solid; hence losing the flag forever and potentially stopping solidification as well. But this should never happen anyway.